apps/test/unit/templates/instructions/CommentAreaTest.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import {UnconnectedCommentArea as CommentArea} from '@cdo/apps/templates/instructions/CommentArea';
+
+const DEFAULT_PROPS = {
+  disabledMode: false,
+  onCommentChange: () => {},
+  comment: 'Good Work!',
+  placeholderText: 'Add your comment here'
+};
+
+describe('CommentArea', () => {
+  it('If displayed mode, textarea is display only', () => {
```

Comment:
Style: It's nice to write your test titles as if the `it(` was part of a complete statement.

```suggestion
  it('renders a readonly textarea when disabledMode=true', () => {
```

If you write test names this way, failures will be reported with a composite test name using the `describe()` so you'd get `CommentArea renders a readonly textarea when disabledMode=true`.